controllers/tasksRequests.js

```diff
@@ -0,0 +1,60 @@
+const taskRequestsModel = require("../models/taskRequests");
+
+const fetchTaskRequests = async (_, res) => {
+  try {
+    const taskRequests = await taskRequestsModel.fetchTaskRequests();
+
+    if (taskRequests && taskRequests.length > 0) {
+      return res.status(200).json({
+        message: "Task requests returned successfully",
+        taskRequests,
+      });
+    }
+
+    return res.status(400).json({
+      message: "Unable to fetch task requests",
+    });
+  } catch (err) {
+    logger.error("Error while fetching task requests", err);
+    throw err;
+  }
+};
+
+const createTaskRequest = async (req, res) => {
+  try {
+    const { taskId, userId } = req.body;
+
+    const taskRequestResponse = await taskRequestsModel.createTaskRequest(taskId, userId);
+
+    if (taskRequestResponse.message.includes("updated")) {
+      return res.status(200).json(taskRequestResponse);
+    }
+
+    if (taskRequestResponse.message.includes("created")) {
+      return res.status(201).json(taskRequestResponse);
+    }
+
+    return res.status(400).json({ message: taskRequestResponse.message });
```

Comment:
If it's a failed request you also can throw an error and let the catch part handle it.